src/ripple/app/misc/detail/WorkFile.h

```diff
@@ -85,7 +85,8 @@ WorkFile::run()
             &WorkFile::run, shared_from_this())));
 
     error_code ec;
-    auto const fileContents = getFileContents(ec, path_);
+    // max file size of 1Mb
```

Comment:
Should be `1MB`

Reply:
Done

Reply:
Even better, now that I'm calling `megabytes(1)`, I can remove the comment entirely. Win!